pkgs/os-specific/linux/libselinux/default.nix

```diff
@@ -23,6 +23,13 @@ stdenv.mkDerivation rec {
 
   postPatch = optionalString enablePython ''
     sed -i -e 's|\$(LIBDIR)/libsepol.a|${libsepol}/lib/libsepol.a|' src/Makefile
+
+    # Workaround for a change in Python import stuff in SWIG 3.0.8.
+    # This has been fixed in SELinux upstream (commit a9604c30) but
```

Comment:
Maybe export the commit patch and use it directly in patches (`git format-patch -1 a9604c30`)? with a comment letting know this is a backport ?

It will be easier for the maintainer.